source/tests/hybrid_tests.cpp

```diff
@@ -34,6 +34,10 @@ TIMEMORY_TEST_DEFAULT_MAIN
 #include "timemory/variadic/auto_hybrid.hpp"
 #include "timemory/variadic/component_hybrid.hpp"
 
+#if TIMEMORY_WINDOWS
+#include "timemory/components/papi/papi_array.hpp"
```

Comment:
Why did you need to add this? Timemory shouldn't need to see the definition of the `papi_array` struct when papi support is disabled.

Reply:
When I do not include this header, I get the following error

```
Build started...
1>------ Build started: Project: hybrid_tests, Configuration: Release x64 ------
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\MSBuild\Microsoft\VC\v160\Microsoft.CppBuild.targets(1069,5): warning MSB8027: Two or more files with the name of extern.cpp will produce outputs to the same location. This can lead to an incorrect build result.  The files involved are C:\Users\e27290\projects\crow-mfw\3rdparty\timemory\source\timemory\manager\extern.cpp, C:\Users\e27290\projects\crow-mfw\3rdparty\timemory\source\timemory\storage\extern.cpp, C:\Users\e27290\projects\crow-mfw\3rdparty\timemory\source\timemory\operations\extern.cpp, C:\Users\e27290\projects\crow-mfw\3rdparty\timemory\source\timemory\config\extern.cpp.
1>Done building project "timemory-io-component-static.vcxproj".
1>hybrid_tests.cpp
1>C:\Users\e27290\projects\crow-mfw\3rdparty\timemory\source\tests\hybrid_tests.cpp(25,9): warning C4068: unknown pragma 'GCC'
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\VC\Tools\MSVC\14.28.29333\include\type_traits(621,41): error C2139: 'tim::component::papi_array<8>': an undefined class is not allowed as an argument to compiler intrinsic type trait '__is_constructible'
1>C:\Users\e27290\projects\crow-mfw\3rdparty\timemory\source\timemory/components/papi/types.hpp(48): message : see declaration of 'tim::component::papi_array<8>'
1>C:\Users\e27290\projects\crow-mfw\3rdparty\timemory\source\timemory/variadic/bundle.cpp(820): message : see reference to class template instantiation 'std::is_constructible<tim::component::papi_array<8>>' being compiled
1>C:\Users\e27290\projects\crow-mfw\3rdparty\timemory\source\timemory/variadic/bundle.hpp(606): message : while compiling class template member function 'std::enable_if<0,bool>::type tim::bundle<Tag,tim::component_bundle<Tag>,tim::variadic::mixed_wrapper_types<std::tuple<tim::component::wall_clock,tim::component::cpu_clock,tim::component::cpu_util,tim::component::peak_rss,tim::component::wall_clock *,tim::component::cpu_clock *,tim::component::cpu_util *,tim::component::peak_rss *,tim::component::page_rss *,tim::component::papi_array<8> *,tim::component::vtune_frame *,tim::component::vtune_event *>>>::init(Args &&...,std::enable_if<!std::is_constructible<T,Args...>::value&&std::is_default_constructible<_Uty2>::value,long>::type)'
1>        with
1>        [
1>            Tag=tim::project::timemory
1>        ]
1>C:\Users\e27290\projects\crow-mfw\3rdparty\timemory\source\tests\hybrid_tests.cpp(210): message : see reference to function template instantiation 'std::array<bool,6> tim::bundle<Tag,tim::component_bundle<Tag>,tim::variadic::mixed_wrapper_types<std::tuple<tim::component::wall_clock,tim::component::cpu_clock,tim::component::cpu_util,tim::component::peak_rss,tim::component::wall_clock *,tim::component::cpu_clock *,tim::component::cpu_util *,tim::component::peak_rss *,tim::component::page_rss *,tim::component::papi_array<8> *,tim::component::vtune_frame *,tim::component::vtune_event *>>>::initialize<tim::component::wall_clock,tim::component::cpu_clock,tim::component::cpu_util,tim::component::peak_rss,tim::component::page_rss,tim::component::papi_array_t,>(void)' being compiled
1>        with
1>        [
1>            Tag=tim::project::timemory
1>        ]
1>C:\Users\e27290\projects\crow-mfw\3rdparty\timemory\source\tests\hybrid_tests.cpp(210): message : see reference to function template instantiation 'std::array<bool,6> tim::bundle<Tag,tim::component_bundle<Tag>,tim::variadic::mixed_wrapper_types<std::tuple<tim::component::wall_clock,tim::component::cpu_clock,tim::component::cpu_util,tim::component::peak_rss,tim::component::wall_clock *,tim::component::cpu_clock *,tim::component::cpu_util *,tim::component::peak_rss *,tim::component::page_rss *,tim::component::papi_array<8> *,tim::component::vtune_frame *,tim::component::vtune_event *>>>::initialize<tim::component::wall_clock,tim::component::cpu_clock,tim::component::cpu_util,tim::component::peak_rss,tim::component::page_rss,tim::component::papi_array_t,>(void)' being compiled
1>        with
1>        [
1>            Tag=tim::project::timemory
1>        ]
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\VC\Tools\MSVC\14.28.29333\include\type_traits(1492): message : see reference to function template instantiation 'void hybrid_tests::SetUp::<lambda_6fceb10cd9f249d2336b41f8337467d9>::operator ()<tim::component_bundle<Tag,tim::component::wall_clock,tim::component::cpu_clock,tim::component::cpu_util,tim::component::peak_rss,tim::component::wall_clock *,tim::component::cpu_clock *,tim::component::cpu_util *,tim::component::peak_rss *,tim::component::page_rss *,tim::component::papi_array<8> *,tim::component::vtune_frame *,tim::component::vtune_event *>>(tim::component_bundle<Tag,tim::component::wall_clock,tim::component::cpu_clock,tim::component::cpu_util,tim::component::peak_rss,tim::component::wall_clock *,tim::component::cpu_clock *,tim::component::cpu_util *,tim::component::peak_rss *,tim::component::page_rss *,tim::component::papi_array<8> *,tim::component::vtune_frame *,tim::component::vtune_event *> &) const' being compiled
1>        with
1>        [
1>            Tag=tim::project::timemory
1>        ]
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\VC\Tools\MSVC\14.28.29333\include\type_traits(1703): message : see reference to alias template instantiation 'std::_Decltype_invoke_nonzero<hybrid_tests::SetUp::<lambda_6fceb10cd9f249d2336b41f8337467d9>&,tim::component_bundle<Tag,tim::component::wall_clock,tim::component::cpu_clock,tim::component::cpu_util,tim::component::peak_rss,tim::component::wall_clock *,tim::component::cpu_clock *,tim::component::cpu_util *,tim::component::peak_rss *,tim::component::page_rss *,tim::component::papi_array<8> *,tim::component::vtune_frame *,tim::component::vtune_event *>&,>' being compiled
1>        with
1>        [
1>            Tag=tim::project::timemory
1>        ]
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\VC\Tools\MSVC\14.28.29333\include\type_traits(1751): message : see reference to alias template instantiation 'std::_Is_invocable_r_<void,hybrid_tests::SetUp::<lambda_6fceb10cd9f249d2336b41f8337467d9>&,tim::component_bundle<Tag,tim::component::wall_clock,tim::component::cpu_clock,tim::component::cpu_util,tim::component::peak_rss,tim::component::wall_clock *,tim::component::cpu_clock *,tim::component::cpu_util *,tim::component::peak_rss *,tim::component::page_rss *,tim::component::papi_array<8> *,tim::component::vtune_frame *,tim::component::vtune_event *>&>' being compiled
1>        with
1>        [
1>            Tag=tim::project::timemory
1>        ]
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\VC\Tools\MSVC\14.28.29333\include\functional(1005): message : see reference to class template instantiation 'std::_Is_invocable_r<_Ret,hybrid_tests::SetUp::<lambda_6fceb10cd9f249d2336b41f8337467d9> &,tim::component_bundle<Tag,tim::component::wall_clock,tim::component::cpu_clock,tim::component::cpu_util,tim::component::peak_rss,tim::component::wall_clock *,tim::component::cpu_clock *,tim::component::cpu_util *,tim::component::peak_rss *,tim::component::page_rss *,tim::component::papi_array<8> *,tim::component::vtune_frame *,tim::component::vtune_event *> &>' being compiled
1>        with
1>        [
1>            _Ret=void,
1>            Tag=tim::project::timemory
1>        ]
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\VC\Tools\MSVC\14.28.29333\include\functional(1005): message : see reference to variable template 'const bool conjunction_v<std::negation<std::is_same<<lambda_6fceb10cd9f249d2336b41f8337467d9>,std::function<void __cdecl(tim::component_bundle<tim::project::timemory,tim::component::wall_clock,tim::component::cpu_clock,tim::component::cpu_util,tim::component::peak_rss,tim::component::wall_clock *,tim::component::cpu_clock *,tim::component::cpu_util *,tim::component::peak_rss *,tim::component::page_rss *,tim::component::papi_array<8> *,tim::component::vtune_frame *,tim::component::vtune_event *> &)> > >,std::_Is_invocable_r<void,<lambda_6fceb10cd9f249d2336b41f8337467d9> &,tim::component_bundle<tim::project::timemory,tim::component::wall_clock,tim::component::cpu_clock,tim::component::cpu_util,tim::component::peak_rss,tim::component::wall_clock *,tim::component::cpu_clock *,tim::component::cpu_util *,tim::component::peak_rss *,tim::component::page_rss *,tim::component::papi_array<8> *,tim::component::vtune_frame *,tim::component::vtune_event *> &> >' being compiled
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\VC\Tools\MSVC\14.28.29333\include\type_traits(621): message : see reference to alias template instantiation 'std::_Func_class<_Ret,tim::component_bundle<Tag,tim::component::wall_clock,tim::component::cpu_clock,tim::component::cpu_util,tim::component::peak_rss,tim::component::wall_clock *,tim::component::cpu_clock *,tim::component::cpu_util *,tim::component::peak_rss *,tim::component::page_rss *,tim::component::papi_array<8> *,tim::component::vtune_frame *,tim::component::vtune_event *> &>::_Enable_if_callable_t<hybrid_tests::SetUp::<lambda_6fceb10cd9f249d2336b41f8337467d9>&,std::function<void (tim::component_bundle<Tag,tim::component::wall_clock,tim::component::cpu_clock,tim::component::cpu_util,tim::component::peak_rss,tim::component::wall_clock *,tim::component::cpu_clock *,tim::component::cpu_util *,tim::component::peak_rss *,tim::component::page_rss *,tim::component::papi_array<8> *,tim::component::vtune_frame *,tim::component::vtune_event *> &)>>' being compiled
1>        with
1>        [
1>            _Ret=void,
1>            Tag=tim::project::timemory
1>        ]
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\VC\Tools\MSVC\14.28.29333\include\type_traits(639,49): error C2139: 'tim::component::papi_array<8>': an undefined class is not allowed as an argument to compiler intrinsic type trait '__is_constructible'
1>C:\Users\e27290\projects\crow-mfw\3rdparty\timemory\source\timemory/components/papi/types.hpp(48): message : see declaration of 'tim::component::papi_array<8>'
1>C:\Users\e27290\projects\crow-mfw\3rdparty\timemory\source\timemory/variadic/bundle.cpp(820): message : see reference to class template instantiation 'std::is_default_constructible<tim::component::papi_array<8>>' being compiled
1>Done building project "hybrid_tests.vcxproj" -- FAILED.
========== Build: 0 succeeded, 1 failed, 0 up-to-date, 0 skipped ==========
```

Reply:
Can you try applying this patch `source/timemory/variadic/bundle.hpp`:

```console
 diff --git a/source/timemory/variadic/bundle.hpp b/source/timemory/variadic/bundle.hpp
 index 4884da3f..5d075019 100644
 --- a/source/timemory/variadic/bundle.hpp
 +++ b/source/timemory/variadic/bundle.hpp
 @@ -318,7 +318,7 @@ public:
      {
          using T = remove_pointer_decay_t<U>;
          return can_stack_init<T>() || can_heap_init<T>() || can_placement_init<T>() ||
 -               has_user_bundle();
 +               (trait::is_available<T>::value && has_user_bundle());
      }
  
      /// Query at compile-time whether initialization will occur on the heap.
 @@ -339,7 +339,8 @@ public:
      static constexpr bool will_opaque_init()
      {
          using T = remove_pointer_decay_t<U>;
 -        return has_user_bundle() && !can_stack_init<T>() && !can_heap_init<T>();
 +        return !can_stack_init<T>() && !can_heap_init<T>() &&
 +               (trait::is_available<T>::value && has_user_bundle());
      }
```

The `enable_if_t<will_heap_init<T>() && !will_opaque_init<T>(), bool>` return "type" __*should*__ cause the compiler to remove the function from overload resolution before it evaluates the `enable_if_t<..., int> = 0` in the args but technically if T is not defined, `std::is_constructible<T, ...>` is an undefined behavior so technically MSVC can do whatever it wants there, including throwing a compiler error. Hopefully though this will fix it.

Reply:
I tried the patch (I entered it manually - is there a way to have git apply a patch like that?) but I still get:

I have an uneasy feeling about the very first warning MSB8027 that "can lead to an incorrect build result."
You are building timemory in a way I am not used to, so I am unsure if this is a real problem or not.

```
Build started...
1>------ Build started: Project: hybrid_tests, Configuration: Release x64 ------
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\MSBuild\Microsoft\VC\v160\Microsoft.CppBuild.targets(1069,5): warning MSB8027: Two or more files with the name of extern.cpp will produce outputs to the same location. This can lead to an incorrect build result.  The files involved are C:\Users\e27290\projects\crow-mfw\3rdparty\timemory\source\timemory\manager\extern.cpp, C:\Users\e27290\projects\crow-mfw\3rdparty\timemory\source\timemory\storage\extern.cpp, C:\Users\e27290\projects\crow-mfw\3rdparty\timemory\source\timemory\operations\extern.cpp, C:\Users\e27290\projects\crow-mfw\3rdparty\timemory\source\timemory\config\extern.cpp.
1>Done building project "timemory-common-static.vcxproj".
1>hybrid_tests.cpp
1>C:\Users\e27290\projects\crow-mfw\3rdparty\timemory\source\tests\hybrid_tests.cpp(25,9): warning C4068: unknown pragma 'GCC'
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\VC\Tools\MSVC\14.28.29333\include\type_traits(621,41): error C2139: 'tim::component::papi_array<8>': an undefined class is not allowed as an argument to compiler intrinsic type trait '__is_constructible'
1>C:\Users\e27290\projects\crow-mfw\3rdparty\timemory\source\timemory/components/papi/types.hpp(48): message : see declaration of 'tim::component::papi_array<8>'
1>C:\Users\e27290\projects\crow-mfw\3rdparty\timemory\source\timemory/variadic/bundle.cpp(820): message : see reference to class template instantiation 'std::is_constructible<tim::component::papi_array<8>>' being compiled
1>C:\Users\e27290\projects\crow-mfw\3rdparty\timemory\source\timemory/variadic/bundle.hpp(607): message : while compiling class template member function 'std::enable_if<0,bool>::type tim::bundle<Tag,tim::component_bundle<Tag>,tim::variadic::mixed_wrapper_types<std::tuple<tim::component::wall_clock,tim::component::cpu_clock,tim::component::cpu_util,tim::component::peak_rss,tim::component::wall_clock *,tim::component::cpu_clock *,tim::component::cpu_util *,tim::component::peak_rss *,tim::component::page_rss *,tim::component::papi_array<8> *,tim::component::vtune_frame *,tim::component::vtune_event *>>>::init(Args &&...,std::enable_if<!std::is_constructible<T,Args...>::value&&std::is_default_constructible<_Uty2>::value,long>::type)'
1>        with
1>        [
1>            Tag=tim::project::timemory
1>        ]
1>C:\Users\e27290\projects\crow-mfw\3rdparty\timemory\source\tests\hybrid_tests.cpp(210): message : see reference to function template instantiation 'std::array<bool,6> tim::bundle<Tag,tim::component_bundle<Tag>,tim::variadic::mixed_wrapper_types<std::tuple<tim::component::wall_clock,tim::component::cpu_clock,tim::component::cpu_util,tim::component::peak_rss,tim::component::wall_clock *,tim::component::cpu_clock *,tim::component::cpu_util *,tim::component::peak_rss *,tim::component::page_rss *,tim::component::papi_array<8> *,tim::component::vtune_frame *,tim::component::vtune_event *>>>::initialize<tim::component::wall_clock,tim::component::cpu_clock,tim::component::cpu_util,tim::component::peak_rss,tim::component::page_rss,tim::component::papi_array_t,>(void)' being compiled
1>        with
1>        [
1>            Tag=tim::project::timemory
1>        ]
1>C:\Users\e27290\projects\crow-mfw\3rdparty\timemory\source\tests\hybrid_tests.cpp(210): message : see reference to function template instantiation 'std::array<bool,6> tim::bundle<Tag,tim::component_bundle<Tag>,tim::variadic::mixed_wrapper_types<std::tuple<tim::component::wall_clock,tim::component::cpu_clock,tim::component::cpu_util,tim::component::peak_rss,tim::component::wall_clock *,tim::component::cpu_clock *,tim::component::cpu_util *,tim::component::peak_rss *,tim::component::page_rss *,tim::component::papi_array<8> *,tim::component::vtune_frame *,tim::component::vtune_event *>>>::initialize<tim::component::wall_clock,tim::component::cpu_clock,tim::component::cpu_util,tim::component::peak_rss,tim::component::page_rss,tim::component::papi_array_t,>(void)' being compiled
1>        with
1>        [
1>            Tag=tim::project::timemory
1>        ]
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\VC\Tools\MSVC\14.28.29333\include\type_traits(1492): message : see reference to function template instantiation 'void hybrid_tests::SetUp::<lambda_6fceb10cd9f249d2336b41f8337467d9>::operator ()<tim::component_bundle<Tag,tim::component::wall_clock,tim::component::cpu_clock,tim::component::cpu_util,tim::component::peak_rss,tim::component::wall_clock *,tim::component::cpu_clock *,tim::component::cpu_util *,tim::component::peak_rss *,tim::component::page_rss *,tim::component::papi_array<8> *,tim::component::vtune_frame *,tim::component::vtune_event *>>(tim::component_bundle<Tag,tim::component::wall_clock,tim::component::cpu_clock,tim::component::cpu_util,tim::component::peak_rss,tim::component::wall_clock *,tim::component::cpu_clock *,tim::component::cpu_util *,tim::component::peak_rss *,tim::component::page_rss *,tim::component::papi_array<8> *,tim::component::vtune_frame *,tim::component::vtune_event *> &) const' being compiled
1>        with
1>        [
1>            Tag=tim::project::timemory
1>        ]
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\VC\Tools\MSVC\14.28.29333\include\type_traits(1703): message : see reference to alias template instantiation 'std::_Decltype_invoke_nonzero<hybrid_tests::SetUp::<lambda_6fceb10cd9f249d2336b41f8337467d9>&,tim::component_bundle<Tag,tim::component::wall_clock,tim::component::cpu_clock,tim::component::cpu_util,tim::component::peak_rss,tim::component::wall_clock *,tim::component::cpu_clock *,tim::component::cpu_util *,tim::component::peak_rss *,tim::component::page_rss *,tim::component::papi_array<8> *,tim::component::vtune_frame *,tim::component::vtune_event *>&,>' being compiled
1>        with
1>        [
1>            Tag=tim::project::timemory
1>        ]
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\VC\Tools\MSVC\14.28.29333\include\type_traits(1751): message : see reference to alias template instantiation 'std::_Is_invocable_r_<void,hybrid_tests::SetUp::<lambda_6fceb10cd9f249d2336b41f8337467d9>&,tim::component_bundle<Tag,tim::component::wall_clock,tim::component::cpu_clock,tim::component::cpu_util,tim::component::peak_rss,tim::component::wall_clock *,tim::component::cpu_clock *,tim::component::cpu_util *,tim::component::peak_rss *,tim::component::page_rss *,tim::component::papi_array<8> *,tim::component::vtune_frame *,tim::component::vtune_event *>&>' being compiled
1>        with
1>        [
1>            Tag=tim::project::timemory
1>        ]
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\VC\Tools\MSVC\14.28.29333\include\functional(1005): message : see reference to class template instantiation 'std::_Is_invocable_r<_Ret,hybrid_tests::SetUp::<lambda_6fceb10cd9f249d2336b41f8337467d9> &,tim::component_bundle<Tag,tim::component::wall_clock,tim::component::cpu_clock,tim::component::cpu_util,tim::component::peak_rss,tim::component::wall_clock *,tim::component::cpu_clock *,tim::component::cpu_util *,tim::component::peak_rss *,tim::component::page_rss *,tim::component::papi_array<8> *,tim::component::vtune_frame *,tim::component::vtune_event *> &>' being compiled
1>        with
1>        [
1>            _Ret=void,
1>            Tag=tim::project::timemory
1>        ]
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\VC\Tools\MSVC\14.28.29333\include\functional(1005): message : see reference to variable template 'const bool conjunction_v<std::negation<std::is_same<<lambda_6fceb10cd9f249d2336b41f8337467d9>,std::function<void __cdecl(tim::component_bundle<tim::project::timemory,tim::component::wall_clock,tim::component::cpu_clock,tim::component::cpu_util,tim::component::peak_rss,tim::component::wall_clock *,tim::component::cpu_clock *,tim::component::cpu_util *,tim::component::peak_rss *,tim::component::page_rss *,tim::component::papi_array<8> *,tim::component::vtune_frame *,tim::component::vtune_event *> &)> > >,std::_Is_invocable_r<void,<lambda_6fceb10cd9f249d2336b41f8337467d9> &,tim::component_bundle<tim::project::timemory,tim::component::wall_clock,tim::component::cpu_clock,tim::component::cpu_util,tim::component::peak_rss,tim::component::wall_clock *,tim::component::cpu_clock *,tim::component::cpu_util *,tim::component::peak_rss *,tim::component::page_rss *,tim::component::papi_array<8> *,tim::component::vtune_frame *,tim::component::vtune_event *> &> >' being compiled
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\VC\Tools\MSVC\14.28.29333\include\type_traits(621): message : see reference to alias template instantiation 'std::_Func_class<_Ret,tim::component_bundle<Tag,tim::component::wall_clock,tim::component::cpu_clock,tim::component::cpu_util,tim::component::peak_rss,tim::component::wall_clock *,tim::component::cpu_clock *,tim::component::cpu_util *,tim::component::peak_rss *,tim::component::page_rss *,tim::component::papi_array<8> *,tim::component::vtune_frame *,tim::component::vtune_event *> &>::_Enable_if_callable_t<hybrid_tests::SetUp::<lambda_6fceb10cd9f249d2336b41f8337467d9>&,std::function<void (tim::component_bundle<Tag,tim::component::wall_clock,tim::component::cpu_clock,tim::component::cpu_util,tim::component::peak_rss,tim::component::wall_clock *,tim::component::cpu_clock *,tim::component::cpu_util *,tim::component::peak_rss *,tim::component::page_rss *,tim::component::papi_array<8> *,tim::component::vtune_frame *,tim::component::vtune_event *> &)>>' being compiled
1>        with
1>        [
1>            _Ret=void,
1>            Tag=tim::project::timemory
1>        ]
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\VC\Tools\MSVC\14.28.29333\include\type_traits(639,49): error C2139: 'tim::component::papi_array<8>': an undefined class is not allowed as an argument to compiler intrinsic type trait '__is_constructible'
1>C:\Users\e27290\projects\crow-mfw\3rdparty\timemory\source\timemory/components/papi/types.hpp(48): message : see declaration of 'tim::component::papi_array<8>'
1>C:\Users\e27290\projects\crow-mfw\3rdparty\timemory\source\timemory/variadic/bundle.cpp(820): message : see reference to class template instantiation 'std::is_default_constructible<tim::component::papi_array<8>>' being compiled
1>Done building project "hybrid_tests.vcxproj" -- FAILED.
========== Build: 0 succeeded, 1 failed, 0 up-to-date, 0 skipped ==========
```